domain.yml

```diff
@@ -264,22 +270,22 @@ templates:
           payload: "/feedback{\"feedback_value\":\"positive\"}"
         - title: "👎"
           payload: "/feedback{\"feedback_value\":\"negative\"}"
-  utter_thank_usecase:
-    - text: "Cool, thank you!"
   utter_canthelp:
     - text: "I'm sorry I couldn't help you. If you have general questions about the Stack, please head to https://forum.rasa.com/, otherwise contact us at hi@rasa.com for anything else."
   utter_nohelp:
     - text: "For now that's all I can do, so feel free to try it out. If you have any suggestions for what I should learn, let me know!"
   utter_thumbsup:
     - text: "👍"
-    - text: "ok"
-    - text: "got it"
+    - text: "Okay."
+    - text: "Got it."
   utter_cantsignup:
     - text: "I can't sign you up if you don't provide your email 😉"
   utter_ask_isbot:
-    - text: "Yep I'm a bot!"
+    - text: "Yep, I'm a bot!"
   utter_cant_answer_tech_question:
-    - text: "I can't answer that"
+    - text: "I can't answer that."
+  utter_no_email:
+    - text: "Hmm, I'm not sure that's a valid email, please make sure to include the full address 😅"
```

Comment:
didn't we already have another utterance for this?

Reply:
No, we just have utter_cant_signup: "I can't sign you up if you don't provide your email 😉" which doesn't really make sense here i think

Reply:
aha got it!